lib/client/api.go

```diff
@@ -946,7 +946,8 @@ func (tc *TeleportClient) Login() error {
 
 	var response *web.SSHLoginResponse
 	switch tc.SecondFactorType {
-	case "hotp":
+	case "otp":
```

Comment:
can you introduce constants in `teleport/constants.go` instead of using inline constants?